test/tools/CMakeLists.txt

```diff
@@ -39,5 +39,35 @@ add_evmc_tool_test(
     "Result: +success[\r\n]+Gas used: +6[\r\n]+Output: +02[\r\n]"
 )
 
+add_evmc_tool_test(
+    invalid_hex_code
+    "--vm $<TARGET_FILE:evmc::example-vm> run 0x600"
+    "code: \\(incomplete hex byte pair\\) OR \\(File does not exist: 0x600\\)"
+)
+
+add_evmc_tool_test(
+    invalid_hex_input
+    "--vm $<TARGET_FILE:evmc::example-vm> run 0x --input aa0y"
```

Comment:
Hmm, we accept this weird Ethereum concept of `0x` for empty input? Does it accept `run ''` or `run ""` ?

Reply:
All should work

Reply:
Could you add one more test with that? I hope we can remove the `run 0x` case at some point in the future :)

Reply:
https://github.com/ethereum/evmc/blob/master/test/unittests/hex_test.cpp#L71

Reply:
I meant adding a test for `run ""`.

And yes, I know it currently accepts `0x` as empty and not asking to remove it right now :)

Reply:
I tried 30 mins, but I don't know how to do this in CMake.

Reply:
Added.

Reply:
Hmm, I thought `"--vm $<TARGET_FILE:evmc::example-vm> run '' --input aa0y"` would have been enough?